fluster/decoders/gstreamer.py

```diff
@@ -60,7 +60,8 @@ def output_format_to_gst(output_format: OutputFormat) -> str:
     """Return GStreamer pixel format"""
     mapping = {
         OutputFormat.GRAY: "GRAY8",
-        OutputFormat.GRAY10LE: "GRAY10_LE32",
+        OutputFormat.GRAY10LE16: "GRAY10_LE16",
+        OutputFormat.GRAY10LE32: "GRAY10_LE32",
```

Comment:
GRAY10LE16 does not exist in GStreamer, please submit upstream so we have matching names. Should be a single plane variant of GST_VIDEO_FORMAT_I420_10LE. It is incompatible with GRAY16_LE because GRAY10LE padding is on the opposite side.